src/pages/pricing/index.tsx

```diff
@@ -80,10 +72,46 @@ const Pricing: FunctionComponent<PricingProps> = () => {
           </div>
         )}
         {!needsEmail && (
-          <SelectPlan prices={data} onClickCheckout={onClickCheckout} />
+          <SelectPlan>
+            <div className="py-8 px-6 text-center bg-gray-50 lg:flex-shrink-0 lg:flex lg:flex-col lg:justify-center lg:p-12">
```

Comment:
the composition makes this files a little chunkier, but also makes it easier to work with since there's not a lot of reason to visit the `SelectPrice` component